models/tagging.rb

```diff
@@ -0,0 +1,19 @@
+require 'active_record'
+
+class Tagging < ActiveRecord::Base
+
+  belongs_to :note
+  belongs_to :tag
+
+  validates :note, presence: true
+  validates :tag, presence: true
+
+
+  def as_json(arg)
```

Comment:
We're probably not ever rendering taggings as json, so this is likely not necessary